sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefServiceTest.java

```diff
@@ -434,21 +434,66 @@ void manageBindingForLDEPF_when_manyIedSourceFound_should_return_report() {
     }
 
     @Test
-    void manageBindingForLDEPF_when_extRefInFlowKindInternalAndExternal_update_successfully_should_return_no_report() {
+    void manageBindingForLDEPF_when_extRefMatchFlowKindInternalOrExternal_should_update_successfully_the_ExtRef_And_DAI_In_RBDR_bRBDR_LNodes_with_no_report() {
         //Given
-        String fileName = "LDEPF_Setting_file.csv";
-        InputStream inputStream = Objects.requireNonNull(CsvUtils.class.getClassLoader().getResourceAsStream(fileName), "Resource not found: " + fileName);
-        InputStreamReader reader = new InputStreamReader(inputStream);
         SCL scd = SclTestMarshaller.getSCLFromFile("/scd-ldepf/scd_ldepf_dataTypeTemplateValid.xml");
-        SettingLDEPFCsvHelper settingLDEPFCsvHelper = new SettingLDEPFCsvHelper(reader);
+
+        TChannel analogueChannel1WithBayInternalScope = new TChannel();
```

Comment:
Je ne sais pas si c'est une bonne idée, mais ne serait-il pas mieux de mettre tous ces paramètres dans un fichier xml puis unmarshall en EPF -> TChannel ? Les tests deviennent plus long maintenant.

Reply:
delete french part in the comment

Reply:
I keep it simple and easy. I provide the manually created object to the service that requires an EPF object.